e2e/parameterTableAndKeyboardShortcuts.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { test, expect } from '@playwright/test';
+
+test('Creating a Simple Graph', async ({ page }) => {
+  
+  await page.goto('http://localhost:8888/');
+
+  // Expect a title "to contain" a substring.
+  await expect(page).toHaveTitle(/EAGLE/);
+
+  //--------ENABLE EXPERT MODE----------
+  //open settings modal via keyboard shortcut
+  await page.press('body','o');
+
+  //enable expert mode
+  const uiModeSelect = await page.getByPlaceholder('uiMode')
+  uiModeSelect.selectOption({value:'Expert'})
+  //close settings modal (wait is needed, bootstrap is not ready to close the modal again that quickly)
+  await page.waitForTimeout(500);
+  await page.getByRole('button', { name: 'OK' }).click()
+
+  //add a helloworld app to the graph by clicking it's icon
+  await page.locator('#addPaletteNodeHelloWorldApp').click();
+  //agree to create a new graph with it's auto-generated name
+  await page.waitForTimeout(500);
+  await page.getByRole('button', { name: 'OK' }).click();
+
+  //-----------EDITING THE NODE IN THE PARAMETERS TABLE--------
+  //open the parameters table via keybaord shortcut
+  await page.press('body','t');
+
+  //add a new parameter
+  await page.getByRole('button', { name: 'Add Parameter' }).click();
+  //select the new parameter
+  await page.getByRole('cell', { name: 'New Parameter' }).getByPlaceholder('New Parameter').click();
+
+  //select text and replace it with a test name
+  await page.getByRole('row', { name: 'New Parameter' }).getByPlaceholder('New Parameter').selectText()
+  await page.getByRole('row', { name: 'New Parameter' }).getByPlaceholder('New Parameter').pressSequentially('test parameter');
+  //check that the name has been changed
+  await expect(page.getByRole('row').last().locator('.column_DisplayText input')).toHaveValue('test parameter')
```

Comment:
This is good. After pressing the 'New parameter' button and entering the new name, you then check that the new parameter appeared as a new row. Nice.